pyrobot/plugins/get_admins.py

```diff
@@ -0,0 +1,37 @@
+import pyrogram
+from pyrogram import Filters, Client
+from pyrobot import app, cmd
```

Comment:
this will fail, right?

Reply:
no it working..

Reply:
there is no "cmd" ?? How will it work? 🙄🙄

Reply:
https://github.com/SpEcHiDe/PyroGramUserBot/pull/5

 fixed

Reply:
why you opening multiple PRs? :thinking:

Reply:
how can I only one pr ?